sklearn/utils/tests/test_extmath.py

```diff
@@ -421,6 +431,93 @@ def test_incremental_variance_update_formulas():
     assert_almost_equal(final_variances, A.var(axis=0), 6)
     assert_almost_equal(final_count, A.shape[0])
 
+    # It is supposed to break, but it does not!
+    # assert_array_almost_equal(A.var(axis=0), one_pass_var(A))
```

Comment:
It don't think it's supposed to break on such a small and well behaved dataset. The page http://www.mathsisfun.com/data/standard-deviation.html does not speak about numerical stability at all.

Let't remove the `one_pass_var` function from this test.